test/common.js

```diff
@@ -29,7 +29,11 @@ exports.createDummyProject = function(dir, name, done) {
 
 exports.resetWorkspace = function(done) {
   async.each(workspace.models(), function(model, cb) {
-    model.destroyAll(cb);
+    if (typeof model.destroyAll === 'function') {
```

Comment:
@ritch why don't all workspace models have a `.destroyAll()` method in loopback-workspace@3.16.x?